packages/data/src/onboarding/deprecated-tasks.js

```diff
@@ -54,7 +54,7 @@ export class DeprecatedTasks {
 						time: task.time,
 						level: task.level ? parseInt( task.level, 10 ) : 3,
 						list_id: task.type || 'extended',
-						is_visible: task.visible,
+						can_view: task.visible,
```

Comment:
I think this won't create any issues since it's passed to the backend, right @louwie17?